src/sections/Home/Playground-home/index.js

```diff
@@ -22,28 +22,28 @@ const ViewsSectionWrapper = styled.div`
     }
 
     .views-section {
-    position: relative;
-    display: flex;
-    flex-direction: row;
-    background-color: ${(props) => props.theme.grey191919ToGreyF2F5F7};
-    margin: 3%;
-    border-radius: 1rem;
-    max-width: 1920px;
-    width: 90vw;
-    height: 500px;
-    justify-content: space-evenly;
-    align-items: center;
-    padding: 0 5% 0 0;
-    box-sizing: border-box;
-    //box-shadow: ${(props) => props.theme.boxShadowBlue477E96};
-    transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
-    overflow: hidden;
+        position: relative;
```

Comment:
remove this unnecessary lints please

Reply:
I believe this indentation is necessary to make it easier to distinguish between the CSS selector and the properties under it. Prior to my change, the selector and properties were all indented on the same level, making it hard to identify nested selectors.